src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -141,6 +144,18 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
         private void OpenInCsvApp(object parameter) {
             CsvAppFileIO.OpenDataCsvApp(DebugEvaluation).DoNotWait();
         }
+
+        /// <summary>
+        /// Deletes variable represented by this mode
+        /// </summary>
+        public void Delete() {
```

Comment:
Let's make this `DeleteAsync`, and use `DoNotWait();` in view (so this method can be tested)